docs/deployment/README.md

```diff
@@ -179,47 +188,41 @@ The name of this iam stack in CloudFormation (e.g. <prefix>-iam).
 
 ###### buckets:
 
-The buckets created in the [Create S3 Buckets](#create-s3-buckets) step.
+The buckets created in the [Create S3 Buckets](#create-s3-buckets) step. Buckets are defined in the config.yml with a key, name, and type. Types should be one of: internal, public, private, or protected. Multiple buckets of each type can be configured.
 
 ------
 
 **Sample new deployment added to config.yml**:
 
-    <iam-deployment-name>:          # e.g. dev (Note: Omit brackets, i.e. NOT <dev>)
+    <iam-deployment-name>:    # e.g. dev (Note: Omit brackets, i.e. NOT <dev>)
       prefix: <stack-prefix>  # prefixes CloudFormation-created iam resources and permissions
       stackName: <stack-name> # name of this iam stack in CloudFormation (e.g. <prefix>-iams)
+      system_bucket: <prefix-internal>
```

Comment:
should this just be <internal>?  I'm not sure what the default bucket names are and before we had <prefix>-internal to represent mhs-internal.  or mhs-cumulus-internal.  Same on L 203

Reply:
and the example/app/config.yml still has `<prefix>-internal`